packages/augur-ui/src/modules/modal/transfer-form.tsx

```diff
@@ -113,14 +121,24 @@ export class TransferForm extends Component<
       fallBackGasCosts,
       GsnEnabled,
       gasPrice,
+      useSigner,
     } = this.props;
     const { relayerGasCosts } = this.state;
     const newAmount = convertExponentialToDecimal(sanitizeArg(amount));
     const bnNewAmount = createBigNumber(newAmount || '0');
     const { errors: updatedErrors, currency } = this.state;
     updatedErrors.amount = '';
-    const availableEth = createBigNumber(balances.eth);
-    const availableDai = createBigNumber(balances.dai);
+    let availableEth = createBigNumber(balances.eth);
+    let availableDai = createBigNumber(balances.dai);
+    const balance = useSigner
+      ? balances[`${currency.toLowerCase()}NonSafe`]
```

Comment:
not a fan of this notation hard to find when searching for usages of loginAccount.balances.XXX

Reply:
will replace 👍